tutorials/algorithms/11_trotterQRTE.ipynb

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
<p>hahaha "which returns a result" is a bit obvious, you could explain what is contained in this result object (evolved state, etc). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1465/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>